back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -15,12 +15,17 @@ def index
     # AdminPublicationsFilteringService is also making a request on projects).
     # But could not find a way to eager-load the polymorphic type in the publication
     # scope.
+
     @projects = Project.where(id:publications.select(:publication_id))
-                       .ordered
                        .includes(:project_images, :phases, :areas, projects_topics: [:topic], admin_publication: [:children])
                        .page(params.dig(:page, :number))
                        .per(params.dig(:page, :size))
 
+    @projects = @projects.ordered unless params[:search].present?
+    @projects = @projects.search_by_all(params[:search]) if params[:search].present?
```

Comment:
Why not:
```suggestion
if params[:search].present?
    @projects = @projects.search_by_all(params[:search])
else 
    @projects = @projects.ordered
end
```
That's a few more lines, but more explicit IMO. If you're into one-liners:
```
projects = params[:search].present? ? projects.search_by_all(params[:search]) : projects.ordered 
```
(removing in passing the instance variable which is probably not needed here).

Reply:
True, it is clearer as an if/else block. I'll change it.